recipes-samples/images/rpb-qt5-image.bb

```diff
@@ -0,0 +1,46 @@
+SUMMARY = "Basic QT5 EGLFS image"
+
+IMAGE_FEATURES += " package-management debug-tweaks ssh-server-openssh hwcodecs tools-debug"
+
+LICENSE = "MIT"
+
+inherit core-image distro_features_check extrausers
+
+# let's make sure we have a good image..
+REQUIRED_DISTRO_FEATURES = "pam systemd opengl"
+
+# by default and without x11-base in IMAGE_FEATURES we default to multi-user.target
+SYSTEMD_DEFAULT_TARGET = "multi-user.target"
+
+CORE_IMAGE_BASE_INSTALL += " \
+    96boards-tools \
+    alsa-utils-aplay \
+    networkmanager \
+    networkmanager-nmtui \
+    coreutils \
+    gps-utils \
+    gpsd \
+    gptfdisk \
+    ${@bb.utils.contains("DISTRO_FEATURES", "x11", "gstreamer1.0-plugins-bad-meta", "", d)} \
```

Comment:
i think we should not need to check for X11 here. if we need , it looks like a bug in oe-core. did you really need it? if so, do you have a log of the failure without it?